src/data/services/EnterpriseSubsidyApiService.js

```diff
@@ -6,8 +6,20 @@ import { configuration } from '../../config';
 class SubsidyApiService {
   static baseUrl = `${configuration.ENTERPRISE_SUBSIDY_BASE_URL}/api/v1`;
 
+  static subsidyV2Url = `${configuration.ENTERPRISE_SUBSIDY_BASE_URL}/api/v2/subsidies`;
```

Comment:
nit: In a similar vein, the other function in this class doesn't define on a constant for the `subsidies` endpoint it uses; I would recommend keeping the endpoint-specific part of the API URL in the specific function itself, or ensure both functions rely on the same constant.